src/converters/RpcConverters.ts

```diff
@@ -6,6 +6,8 @@ import {
     INullableTimestamp
 } from '../../azure-functions-language-worker-protobuf/src/rpc';
 import { InternalException } from "../utils/InternalException";
+import { TraceContext } from '../public/Interfaces';
+import { stringify } from 'querystring';
```

Comment:
what's this import?

Reply:
Weird, I can't remember why/how that's there.. Anyway, removed it.